logging/user.env

```diff
@@ -108,6 +108,21 @@
 # NODE_NAME=mynode.example.com
 
 
+# Use port-forwarding during deployment
+# ------------------------------
+# By default, the deployment process will use
+# the public URLs when accessing the APIs for
+# Kibana and Elasticsearch.  However, in some
+# environments, access to the applications 
+# via these URLs is NOT possible from the
+# machine on which the deployment process will
+# be run.  In those situations, set the following
+# property to "true" to force the deployment
+# scripts to use Kubernetes port-forwarding
+# when accessing the Kibana and/or Elasticsearch
+# APIs.
```

Comment:
Could this be shorter? Maybe just link to the Troubleshooting doc section?

Reply:
I made it shorter and referenced the Troubleshooting guide issue.  But, since this is just a text file (and NOT markdown), I did not include an actual link.  We could include a "real" URL (pointing back to GitHub?) but it is long and messy.  Take a look and see what you think.

Reply:
I meant just include a URL like we do in our sample YAML files, but it's fine